src/table.ts

```diff
@@ -23,44 +23,35 @@ import * as through from 'through2';
 import {Operation as GaxOperation} from 'google-gax';
 import {Json} from './codec';
 import {Database} from './database';
-import {SpannerClient as s} from './v1';
 import {PartialResultStream, Row} from './partial-result-stream';
-import {ReadRequest, TimestampBounds} from './transaction';
+import {
+  ReadRequest,
+  TimestampBounds,
+  CommitResponse,
+  ReadResponse,
+  ReadCallback,
+  Transaction,
+  CommitCallback,
+} from './transaction';
 import {google as databaseAdmin} from '../proto/spanner_database_admin';
-import {Schema, ResourceCallback} from './common';
+import {Schema, ResourceCallback, LongRunningCallback} from './common';
```

Comment:
Seems if you utilize `UpdataSchemaCallback/Response`, you do not need to import `ResourceCallback` here